### ISSUE TITLE
Contribution guidelines shouldn't apply to repo/project names

### ISSUE THREAD

**Issue by Evidlo** [reHackable/awesome-reMarkable#47]
The contribution guidelines are good, but it doesn't make sense for them to apply to the repo names.  In Python, modules are case sensitive and usually lowercase.  This is how my projects appear on PyPi (`remarkable_mouse`, `remarkable_keyboard`, `remarkable_entware`, `remarkable_pdflets`).

It's also unclear how to apply this rule to projects like `remt`, `rmapi`, `rmsync`, `rmtabletdriver`, etc.

I recommend we restrict these guidelines to the description for each submission and let the authors decide how their project is stylized.

**Comment by Ameb** [comment]
> - Maintain consistent and correct naming (ex: `reMarkable`, not `remarkable` or `ReMarkable`).

Maybe modify this lane so it is more clear that the naming conventions apply to the name given on the awesome list and it doesn't need to match the repo/library name exactly. Check https://github.com/reHackable/awesome-reMarkable/pull/42#issuecomment-569768805

**Comment by Evidlo** [comment]
Yes I saw that and I'm trying to make a case why that's the wrong decision.

This isn't a "naming convention" per se because the project names are set
prior to submission.  Project names already vary in so many other ways
(e.g. hyphens/underscores, acronym/no acronym, spaces) that this rule
doesn't help with consistency and has the disadvantage of making the names
wrong.

**Comment by CTXz** [comment]
The initial thought behind this guideline was less practical and more or less an aesthetic decision, however, given that there are already numerous other naming inconsistencies, some of which you've already pointed out, and realizing that the practical benefits indeed outweigh the aesthetic benefits, I suppose you're right.

Unless other members of the reHackable group disagree with me, I suggest I'll edit the list once I find some time (and honestly motivation) to do so. PR's would obviously be appreciated as well.

**Comment by Ameb** [comment]
> Hm.  I'd be willing to go through all of the projects and make sure their names match the authors' official capitalizations.  Would you object to a PR with just those changes?

_Originally posted by @asciiphil in https://github.com/reHackable/awesome-reMarkable/issues/127#issuecomment-829214801_

Let's track it here
